pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -438,3 +438,37 @@ def test_timestamp_multiindex_indexer():
     )
     should_be = pd.Series(data=np.arange(24, len(qidx) + 24), index=qidx, name="foo")
     tm.assert_series_equal(result, should_be)
+
+
+def test_multiindex_loc_order():
```

Comment:
can you move these to pandas/tests/test_multilevel.py

Reply:
Yeah sure.